src/Form/SettingsForm.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\os2forms_dig_sig_server\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Signing settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+  use StringTranslationTrait;
+
+  /**
+   * Name of the config.
+   *
+   * @var string
+   */
+  public static $configName = 'os2forms_dig_sig_server.settings';
```

Comment:
Any specific reasons for not using a `const` here?

```suggestion
  public const string CONFIG_NAME = 'os2forms_dig_sig_server.settings';
```

Reply:
changed to constant, by omit the `string` type to make PHP8.2 compatible.
Typed constants are only supported since PHP8.3.